packages/kafka-iam-auth/src/index.ts

```diff
@@ -157,11 +156,8 @@ const initConsumer = async (
         await consumerHandler(payload);
         await kafkaCommitMessageOffsets(consumer, payload);
       } catch (e) {
-        throw kafkaMessageProcessError(
-          payload.topic,
-          payload.partition,
-          payload.message.offset,
-          e
+        genericLogger.error(
```

Comment:
the exit with error was the intended behaviour so far to avoid losing events.

With this change don't we open to the possibility of skipping messages?
For example
1. message 1 is received
2. `consumerHandler` fails for message 1
3. message 1 failure is logged
4. message 2 is received
5. `consumerHandler` succeed for message 2
6. message 2 is committed

and message 1 is never processed.
Am I missing something?

Reply:
It should retry the message, I'm currently doing other tests to check this condition